### ISSUE TITLE
No coordinates generated when training

### ISSUE THREAD

**Issue by KopiSoftware** [zju3dv/NeuralRecon#35]
Excellent work!
I had a problem when generating results with intel realsense d435i. Both color and depth image has a resolution of 640x480, and the poses are generated by VINS or lidar slam. I collect some simple data in an empty room and try to generate tsdf map with your pretrained midel, but it reports: ' no valid voxel when computing loss' and generated just a few little tiny fragments, which is far from the truth. I managed to produce a ground truth ply file with your tools and found my data was okay, which mean it can correctly produce tsdf map with color image, depth image and poses. So here comes the question: 'is the model over fitted with your experimental device'? I saw the relative issue. I followed the keyframe selection strategy: select rotation above15 degrees and t is over 0.1m, but it don't help anymore. The coarse-to-fine layer produces coordinates but global fuse is not activated. Can you help me with these problems? Thanks to your attention.

**Comment by JiamingSuen** [comment]
> is the model over fitted with your experimental device'?

I don't think so. We've tested multiple cameras with the ScanNet model and the results are normally quite good.

> I followed the keyframe selection strategy: select rotation above15 degrees and t is over 0.1m, but it don't help anymore.

According to your description, many things can go wrong with the custom camera hardware and poses. I can't come up with any potential reason with the given information but I'm quite sure that NeuralRecon can operate normally with RealSense cameras if everything is correctly configured. I recommend further checking the accuracy of the camera poses computed from VINS or LiDAR system. You can also try to capture some data with ARKit first following the tutorial to make sure NeuralRecon can fit your needs.
